support/sdk-test/README.md

```diff
@@ -24,23 +24,23 @@ Refer to the main documentation for information on how to configure the system t
 
 ## Testing
 
-Export the path to the rpms for your chosen platform
+Export the absolute path to the rpms for your chosen platform
 
 ```bash
-export DEPLOY_DIR=./build-qemux86-64/build/tmp/deploy
+export DEPLOY_DIR="$(pwd)/build-qemux86-64/build/tmp/deploy"
 ```
 
 You can start the package-repo container and get a bash prompt in the sdk container with the following `podman-compose` command
 
 ```bash
-podman-compose -f support/sdk-test/compose.yml run sdk-test /bin/bash
+podman-compose -f support/sdk-test/compose.yml run sdk /bin/bash
```

Comment:
does this change work? We are trying to tell the compose to run the sdk container in the compose file with a /bin/bash

Reply:
Ha, reading inverted.